spec/phase2.html

```diff
@@ -297,6 +320,766 @@
+        <td>atk_value_set_value</td>
+        </tr>
+        <tr>
+        <td>1</td>
+        <td>SetTextValue</td>
+        <td>set AXValue</td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        <br>IValueProvider::SetValue</td>
+        <td>ACTION_SET_TEXT</td>
+        <td>atk_editable_text_set_text_contents</td>
+        </tr>
+        <tr>
+        <td>2</td>
+        <td>Cancel</td>
+        <td>AXCancelAction</td>
+        <td>accessibilityPerformEscape</td>
+        <td>
+        </td>
+        <td>IWindowProvider::Close</td>
+        <td>ACTION_DISMISS</td>
+        <td>
+        </td>
+        </tr>
+        <tr>
+        <td>2</td>
+        <td>Collapse</td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>IExpandCollapseProvider::Collapse</td>
+        <td>ACTION_COLLAPSE</td>
+        <td>
+        </td>
+        </tr>
+        <tr>
+        <td>2</td>
+        <td>CopyText</td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>IAccessibleEditableText::copyText</td>
+        <td>
+        </td>
+        <td>ACTION_COPY</td>
+        <td>atk_editable_text_copy_text</td>
+        </tr>
+        <tr>
+        <td>2</td>
+        <td>CutText</td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>IAccessibleEditableText::cutText</td>
+        <td>
+        </td>
+        <td>ACTION_CUT</td>
+        <td>atk_editable_text_cut_text</td>
+        </tr>
+        <tr>
+        <td>2</td>
+        <td>Decrement</td>
+        <td>AXDecrementAction</td>
+        <td>accessibilityDecrement</td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        </tr>
+        <tr>
+        <td>2</td>
+        <td>Expand</td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>IExpandCollapseProvider::Expand</td>
+        <td>ACTION_EXPAND</td>
+        <td>
+        </td>
+        </tr>
+        <tr>
+        <td>2</td>
+        <td>Increment</td>
+        <td>AXIncrementAction</td>
+        <td>accessibilityIncrement</td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>
+        </td>
+        </tr>
+        <tr>
+        <td>2</td>
+        <td>InsertText</td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>IAccessibleEditableText::insertText</td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>atk_editable_text_insert_text</td>
+        </tr>
+        <tr>
+        <td>2</td>
+        <td>PasteText</td>
+        <td>
+        </td>
+        <td>
+        </td>
+        <td>IAccessibleEditableText::pasteText</td>
+        <td>
+        </td>
+        <td>ACTION_PASTE</td>
+        <td>atk_editable_text_paste_text</td>
+        </tr>
+        <tr>
+        <td>2</td>
+        <td>ReplaceText</td>
```

Comment:
In ATK case it may be a combination of select and paste actions.

I'm not sure if we consider the actions as they serve the dual propose, i.e. can be used by users and can be defined by users for custom accessibility. So in the first case, it might be nice to have this action. For the latter case, it's overhead for the users for sure.